amico/models.py

```diff
@@ -222,31 +228,37 @@ def generate( self, out_path, aux, idx_in, idx_out ) :
             progress.update()
 
 
-    def resample( self, in_path, idx_out, Ylm_out ) :
+    def resample( self, in_path, idx_out, Ylm_out, doMergeB0 ) :
```

Comment:
This model is also used in COMMIT, so in case we have to update COMMIT's code as well to account for the modified signature of this `resample` function call.